helpers/control.py

```diff
@@ -114,6 +115,7 @@ def set_user_agent(self, ua):
         self.options.append('-H \'User-Agent: %s\'' % ua)
 
 
+@util.deprecated("framework.Wrk")
```

Comment:
As an alternative name, `deprecated_in_favor_of` could be used. That will make declarations look like a sentences:
```python
@deprecated_in_favor_of("framework.Wrk")
class Wrk(Client):
```

Reply:
Oh, that's too long name

Reply:
It's not that I insist on renaming. Just thought that longer descriptive name will be more... descriptive. But the current name is still fine.